roottest/root/io/uniquePointer/CMakeLists.txt

```diff
@@ -21,6 +21,7 @@ ROOTTEST_COMPILE_MACRO(readPtrColls.C
 
 ROOTTEST_ADD_TEST(readPtrColls
                   MACRO readPtrColls.C+
+                  DEPENDS writePtrColls
```

Comment:
This should not be needed, we use fixtures for this. When I looked at the logs, writePtrColls was nicely executed at the right place...

Reply:
Yes, in most of the places one should replace `DEPENDS` by fixtures.